<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberForm</class>
 <widget class="QWidget" name="MemberForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainGridLayout">
   <item row="2" column="0">
    <widget class="QRadioButton" name="tcpAudioButton">
     <property name="text">
      <string>TCP Audio</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="encryptEnabledBox">
     <property name="text">
      <string>Enable Encryption</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="redundancyLabel">
       <property name="text">
        <string>Redundancy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="redundancyChoice">
       <property name="toolTip">
        <string>One fewer than the number of consecutive dropped packets allowed</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="udpAudioButton">
     <property name="text">
      <string>UDP Audio</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="SectionArea">
     <item>
      <widget class="QLabel" name="sectionLabel">
       <property name="text">
        <string>Section</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sectionChoice"/>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="startJackTripButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start JackTrip</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="jtSelfLoopbackBox">
     <property name="text">
      <string>JackTrip Self Loopback</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="JackParameterForm" name="jackServer" native="true"/>
   </item>
   <item row="2" column="1" rowspan="6">
    <layout class="QVBoxLayout" name="chatArea"/>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="NameArea">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameChoice">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="sentChannelsLabel">
       <property name="text">
        <string>Sent Channels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sentChannelsChoice">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JackParameterForm</class>
   <extends>QWidget</extends>
   <header>jackparameterform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
